### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,25 +3,53 @@
...
 
+- Installed bcftools/norm [#127](https://github.com/genomic-medicine-sweden/tomte/pull/127)
+- Installed bcftools/annotate [#127](https://github.com/genomic-medicine-sweden/tomte/pull/127)
+
 ### `Fixed`
 
+- One line per call on vcf to make vcf suitable for Scout [#127](https://github.com/genomic-medicine-sweden/tomte/pull/127)
+- Added variant caller to vcf to make vcf suitable for Scout [#127](https://github.com/genomic-medicine-sweden/tomte/pull/127)
+- Normalised calls in vcf [#127](https://github.com/genomic-medicine-sweden/tomte/pull/127)
+- Removed regions parameter from ASEReadCounter to obtain ASE for all regions [#128](https://github.com/genomic-medicine-sweden/tomte/pull/128)
```
Comment (fellen31) [inline CHANGELOG.md:18]: This points to the issue, you might want to point to the PR

```suggestion
- Removed regions parameter from ASEReadCounter to obtain ASE for all regions [#129](https://github.com/genomic-medicine-sweden/tomte/pull/129)
```

**Review by fellen31 — COMMENTED** [review]
Nice work!

File: CHANGELOG.md (hunk 1 above)
Comment (Lucpen) [reply CHANGELOG.md:18]: Thanks!

File: nextflow.config [hunk 2]
```diff
@@ -72,6 +72,7 @@ params {
     hook_url                   = null
     help                       = false
     version                    = false
+    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/'
```
Comment (Lucpen) [inline nextflow.config:75]:
```suggestion
    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/raredisease'
```

[the remark this answers is not in the document]
Comment (Lucpen) [reply nextflow.config:75]: Good point, part of the test data is within the rare-disease so I will point there

File: nextflow.config [hunk 3]
```diff
@@ -107,103 +108,109 @@ try {
...
-// try {
-//   includeConfig "${params.custom_config_base}/pipeline/tomte.config"
-// } catch (Exception e) {
-//   System.err.println("WARNING: Could not load nf-core/config/tomte profiles: ${params.custom_config_base}/pipeline/tomte.config")
-// }
+try {
+    includeConfig "${params.custom_config_base}/pipeline/tomte.config"
+} catch (Exception e) {
+    System.err.println("WARNING: Could not load nf-core/config/tomte profiles: ${params.custom_config_base}/pipeline/tomte.config")
+}
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply nextflow.config:115]: You are absolutely sire

Comment (Lucpen) [inline nextflow.config:115]:
```suggestion
// Warning: Uncomment only if a pipeline-specific institutional config already exists on nf-core/configs!
 // try {
 //   includeConfig "${params.custom_config_base}/pipeline/tomte.config"
 // } catch (Exception e) {
 //   System.err.println("WARNING: Could not load nf-core/config/tomte profiles: ${params.custom_config_base}/pipeline/tomte.config")
```

File: nextflow_schema.json [hunk 4]
```diff
@@ -530,6 +530,13 @@
                     "description": "Validation of parameters in lenient more.",
                     "hidden": true,
                     "help_text": "Allows string values that are parseable as numbers or booleans. For further information see [JSONSchema docs](https://github.com/everit-org/json-schema#lenient-mode)."
+                },
+                "pipelines_testdata_base_path": {
+                    "type": "string",
+                    "fa_icon": "far fa-check-circle",
+                    "description": "Base URL or local path to location of pipeline test dataset files",
+                    "default": "https://raw.githubusercontent.com/nf-core/test-datasets/",
```
Comment (Lucpen) [inline nextflow_schema.json:538]:
```suggestion
                    "default": "https://raw.githubusercontent.com/nf-core/test-datasets/raredisease",
```

File: subworkflows/local/call_variants.nf [hunk 5]
```diff
@@ -60,10 +66,39 @@ workflow CALL_VARIANTS {
                 exit 1, "Unknown variantcaller: ${variant_caller}"
         }
 
+        ch_in_split_multi = ch_vcf.join(ch_tbi)
+        SPLIT_MULTIALLELICS(ch_in_split_multi, ch_fasta)
+
+        ch_remove_dup_in = SPLIT_MULTIALLELICS.out.vcf.join(SPLIT_MULTIALLELICS.out.tbi)
+        REMOVE_DUPLICATES(ch_remove_dup_in, ch_fasta)
+
+        ch_genome_chrsizes.flatten().map{chromsizes ->
```
Comment (jemten) [inline subworkflows/local/call_variants.nf:75]:
```suggestion
        ch_genome_chrsizes.flatten().map { chromsizes ->
```

File: subworkflows/local/call_variants.nf [hunk 6]
```diff
@@ -60,10 +66,39 @@ workflow CALL_VARIANTS {
...
+            return [[id:variant_caller], chromsizes]
+            }
+            .set { ch_varcallerinfo }
+
+        ADD_VARCALLER_TO_BED(ch_varcallerinfo)
+        ch_bed_in_annot=ADD_VARCALLER_TO_BED.out.gz.collect()
+        ch_tbi_in_annot=ADD_VARCALLER_TO_BED.out.tbi.collect()
```
Comment (jemten) [inline subworkflows/local/call_variants.nf:82]:
```suggestion
        ch_bed_in_annot = ADD_VARCALLER_TO_BED.out.gz.collect()
        ch_tbi_in_annot = ADD_VARCALLER_TO_BED.out.tbi.collect()
```

**Review by jemten — APPROVED** [review]
Looks good to me. Some formatting notes for readability

File: nextflow.config [hunk 7]
```diff
@@ -108,102 +109,108 @@ try {
...
 // Warning: Uncomment only if a pipeline-specific institutional config already exists on nf-core/configs!
-// try {
-//   includeConfig "${params.custom_config_base}/pipeline/tomte.config"
-// } catch (Exception e) {
-//   System.err.println("WARNING: Could not load nf-core/config/tomte profiles: ${params.custom_config_base}/pipeline/tomte.config")
-// }
+ // try {
+ //   includeConfig "${params.custom_config_base}/pipeline/tomte.config"
+ // } catch (Exception e) {
+ //   System.err.println("WARNING: Could not load nf-core/config/tomte profiles: ${params.custom_config_base}/pipeline/tomte.config")
```
Comment (Lucpen) [inline nextflow.config:115]:
```suggestion
// try {
//   includeConfig "${params.custom_config_base}/pipeline/tomte.config"
// } catch (Exception e) {
//   System.err.println("WARNING: Could not load nf-core/config/tomte profiles: ${params.custom_config_base}/pipeline/tomte.config")
```

File: nextflow_schema.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (Lucpen) [reply nextflow_schema.json]: Done!

**Pull request merged**